src/supertux/gameconfig.cpp

```diff
@@ -221,6 +226,8 @@ Config::load()
     interface_colors_mapping->get("menuroundness", menuroundness, 16.f);
   }
 
+  config_mapping.get("menu_slider_steps", menu_slider_steps);
```

Comment:
IMO this is better, because it allows the user to configure it.

Reply:
Not all sliders represent the same unit. Having the same step for all sliders feels like it would lead to several weird quirks.

If you want your volume slider to go 5 by 5, you set the step to 5, but then if you want to use the mobile controls scale slider, you'd become frustrated from the fact that the step over/undershoots. Mobile controls scale is a more precise value than volume.

It's also not intuitive as you think. You have to go back to the settings menu, then go back to what you were configuring.